portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro

```diff
@@ -209,4 +209,34 @@ definition {
 		}
 	}
 
+	macro viewUserWorkload {
+		key_assignee = "${assignee}";
```

Comment:
Hi @brookedalton 
Just to keep consistency with the other variables, I believe it would be better to add "var" before key_assignee.